assets/stories/skate-components/circleButton.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import type { Meta, StoryObj } from "@storybook/react"
+import React from "react"
+
+import { CircleButton } from "../../src/components/circleButton"
+import { UserAvatar } from "../../src/components/userAvatar"
+
+const meta = {
+  component: CircleButton,
+} satisfies Meta<typeof CircleButton>
+
+export default meta
+type Story = StoryObj<typeof CircleButton>
+
+export const Default: Story = {
+  args: { isActive: false },
```

Comment:
I originally wrote my comment (on the wrong story) when the `children` component was exposed as an argument, which made the arguments table kind of a mess. I don't feel as strongly about it right now, but there are two things about the UX here that kind of throw me off, and I imagine that they'd throw a non-technical user off as well.

1. The default story is something you'd never actually see in production. Right now, the only thing we'll ever see live is a `CircleButton` with a `UserAvatar` inside, and even in the future, we'll always put _something_ inside.
2. There's no way to change the letter rendered inside the icon on this page. Not the biggest deal on this page, because of the `UserAvatar` stories, but I also don't think it's the worst thing in the world to allow this to be more customizable.

Reply:
To take my two piddly comments above, and cast them in a wildly overblown way, both of ☝️ points reflect the question of whether Storybook is a tool for dev, or a tool for people other than dev.

If we place boundaries around what types of things we allow in our Storybook that really only make sense if you know the inner workings of those icons (for instance, the fact that the user icon thing is implemented using a `children` attribute), then that places Storybook as a dev-only tool. 

Another way of putting it is that if an object has any sort of nontrivial interface, then either the story code needs to be hyper-aware of that nontrivial interface (by using a decorator), or the person clicking around in Storybook needs to be aware (by seeing an interface that either lacks options or displays things about JSXNodes). I would prefer to have the Story code itself be the thing that's aware, and let people interacting with Storybook have a better experience.

Reply:
So for what it's worth I was originally conceiving of this all as one component but after talking to design I realized that they actually saw it as two different components, so I think in this case it does make sense to break it apart like this (though I agree in general that UX of Storybook for non-devs should at least be a consideration).

Reply:
Huh, fair enough.

Reply:
Maybe we should start organizing stories according to [atomic design](https://bradfrost.com/blog/post/atomic-web-design/)?
This seems like it'd be a molecule?

Reply:
It'd maybe make it more clear on how granular a component is if it were organized according to atomic design, which I see as addressing point 1

> 1. The default story is something you'd never actually see in production. Right now, the only thing we'll ever see live is a `CircleButton` with a `UserAvatar` inside, and even in the future, we'll always put _something_ inside.

Reply:
If we feel that the solution is to pursue a large reorganization of our Storybook stories according to atomic design, is that something we can look into in a followup task?

Reply:
@lemald https://app.asana.com/0/1148853526253420/1206580891011553/f

Reply:
Thanks!